src/Orchard.Web/Core/Title/Placement.info

```diff
@@ -7,7 +7,7 @@
     -->
     <Place Parts_Title_Edit="Content:before.5"/>
     <Match DisplayType="Detail">
-        <Place Parts_Title="Header:5"/>
```

Comment:
Why change the default zone of the Title ? Not related to this PR I think, and quite a breaking change for themes.

Reply:
oops... We were testing things and forgot to revert this. doing it.